python/interpret_community/common/explanation_utils.py

```diff
@@ -149,21 +149,6 @@ def _is_one_to_many(feature_map):
     return ((feature_map > 0).sum(axis=0) >= 2).sum() == 0
 
 
-def _transform_data(data, data_mapper=None):
```

Comment:
_transform_data is used in scoring explainers

Reply:
Thanks for pointing this. Since this function is not used in interpret-community, I have ported the code to azureml-interpret. This can be cleaned up now.

Reply:
sounds great!